dev/com.ibm.ws.transaction/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -78,3 +78,12 @@ manual=Manual
 
 default.max.shutdown.delay=Default maximum shutdown delay
 default.max.shutdown.delay.desc=Default maximum shutdown delay
+
+lease.check.strategy=Lease check strategy
+lease.check.strategy.desc=The algorithm used by the recovery service to check the recovery lease log
+
+lease.check.interval=Lease check interval
+lease.check.interval.desc=The interval after which the lease check strategy is executed
```

Comment:
The interval after which the lease check strategy is executed

IBM terminology advises to avoid using "execute" where possible. Would it be  correct to use "implemented" instead?

The interval after which the lease check strategy is implemented

Reply:
Please use "run" rather than "executed"